crypto/mem_clr.c

```diff
@@ -60,6 +60,7 @@
 #include <string.h>
 #include <openssl/crypto.h>
 
+extern unsigned char cleanse_ctr;
```

Comment:
This is the wrong way to fix it! The whole point of requiring a previous declaration is to ensure everyone uses the same type - i.e. the extern should be in an appropriate header.

Putting the extern in the same source unit is never the correct fix.

In this case, looks like it should go in some shared internal header.